webpack.config.js

```diff
@@ -1,19 +1,20 @@
-const path = require('path')
-const webpack = require('webpack')
-const BundleAnalyzerPlugin = require('webpack-bundle-analyzer').BundleAnalyzerPlugin;
+const path = require("path");
```

Comment:
This was made by `prettier`. I noticed that some parts of the code are using `prettier` but others aren't. What's the distinction that I should make with this? Does it make sense to do a commit to `master` that `prettier`ify's everything?